gradle/lint.gradle

```diff
@@ -3,6 +3,6 @@ android {
         checkReleaseBuilds false
         warningsAsErrors true
         xmlReport false
-        disable 'AllowBackup'
+        disable 'AllowBackup', 'UnusedResources'
```

Comment:
I guess we want to warn unusedResources?

Reply:
agree with @pengdev here unless there is some very strong reason 😄

Reply:
correct. there are some unused resources. i will try to remove those.

Reply:
We have quite a few unused resources and some resources that are defined in one module and used in another module. since we run lint on every module, it couldn't figure out if these resources are used in other modules as well. I have created an internal ticket to resolve this, we can do that after this pr is merged. wdyt cc @kiryldz @pengdev ?

Reply:
Yes, since there's a lot of warnings across the codebase, let's disable `'UnusedResources'` and ticket out the tailwork to optimise separately.